R/DataSetList.R

```diff
@@ -538,7 +552,18 @@ get_dim <- function(dsList) {
 #' @examples
 #' get_funcId(dsl)
 get_funcId <- function(dsList) {
-  sort(unique(sapply(dsList, function(d) attr(d, 'funcId'))))
+  ll <- unique(unname(unlist(sapply(dsList, function(d) attr(d, 'funcId')))));
+  if(is.integer(ll)) {
+    return(sort(ll));
```

Comment:
I will drop the semicolumns when merging into master ;)

Reply:
You can drop them, no problem.

But since I am a Java person, I must must must put semicolons wherever the syntax allows. I cannot resist. MUST PUT SEMICOLON. SEMICOLON!!!1!11!!!! :-D :-D

Reply:
> You can drop them, no problem.
> 
> But since I am a Java person, I must must must put semicolons wherever the syntax allows. I cannot resist. MUST PUT SEMICOLON. SEMICOLON!!!1!11!!!! :-D :-D

Haha, fully understand;)